modules/virt-removing-vm-delete-protection.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * virt/managing-vms/virt-enabling-disabling-vm-delete-protection.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-removing-vm-delete-protection_{context}"]
+
+= Removing the virtual machine delete protection option
+
+When you enable delete protection on a VM (virtual machine), you ensure that the VM cannot be inadvertently deleted. You can also choose to disable the protection for a VM.
+
+As a cluster administrator, you can choose not to make the VM delete protection option available. VMs with delete protection already enabled retain that setting; for any new VMs that are created, enabling the option is not allowed.
```

Comment:
"choose not to make" sounds a little wordy. Maybe something like this?

Also a suggestion for the second sentence.

```suggestion
As a cluster administrator, you can remove the VM delete protection option completely. VMs with delete protection already enabled retain that setting; any new VMs that are created do not have delete protection as an option.
```

Reply:
@aspauldi Looks like this comment was missed.